src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -129,17 +132,22 @@ const formConfig = {
       title: 'Signer information',
       pages: {
         formSignature: {
+          // initialData: mockdata.data,
           path: 'form-signature',
           title: 'Form signature',
           ...formSignatureSchema,
         },
+        signerEmail: {
+          path: 'signer-email',
+          title: 'Your email Address',
```

Comment:
Should the "A" be lower case here?

Reply:
Yes....yes it should. Somehow several of us missed this one. So sorry, that's my bad! I've fixed it. Let me know if you need me to get another team review for it. :)